sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -191,41 +188,19 @@ export class InstitutionController extends BaseController {
     const institution = await this.institutionService.getInstituteByUserName(
       user.userName,
     );
-    const institutionUserAndCount = await this.institutionService.allUsers(
+    if (!institution) {
```

Comment:
I believe this check is not needed here, as we are using the decorator IsInstitutionAdmin() which validates it already.

Reply:
`IsInstitutionAdmin()` is only checking for admin roles 
![image](https://user-images.githubusercontent.com/77353155/146423374-699fbb47-f7e9-49a1-bb8f-5c39c085b753.png). And i see most of the controller using this decorate has this check, 
![image](https://user-images.githubusercontent.com/77353155/146423538-9256ba92-27db-4b08-a3b3-445bc5f5a972.png)
, let me know if I am missing anything